.github/workflows/submit.yml

```diff
@@ -0,0 +1,46 @@
+name: Kodi Addon-Submitter
+
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  kodi-addon-submitter:
+    runs-on: ubuntu-latest
+    name: Kodi addon submitter
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Generate distribution zip and submit to official kodi repository
+        id: kodi-addon-submitter
+        uses: xbmc/action-kodi-addon-submitter@v1.1
+        with:
+          kodi-repository: repo-scripts
+          kodi-version: matrix
+          addon-id: script.service.checkpreviousepisode
+          kodi-matrix: true # Submit a Python 2/3 compatible addon to matrix branch in addition to the target branch
+        env: # Make sure you create the below secrets (GH_TOKEN and EMAIL)
+          GH_USERNAME: ${{ github.actor }}
+          GH_TOKEN: ${{secrets.GH_TOKEN}}
```

Comment:
You will need to set this up for this repo, and add one of your personal tokens which you can create in your profile settings